.github/workflows/tableau-connector.yml

```diff
@@ -4,6 +4,16 @@ on:
     branches: [ main, develop ]
   pull_request:
     branches: [ main, develop ]
+  workflow_dispatch:
+    inputs:
+      sign:
+        description: Build and Sign Artifacts (for release only)
+        type: boolean
+        required: false
+        default: false
+env:
+  #SIGNING_ENABLED: ${{ github.event.inputs.sign }}
+  SIGNING_ENABLED: true
```

Comment:
Before changes are merged this should be reverted.

Reply:
removed